lib/web_ui/lib/pointer.dart

```diff
@@ -153,7 +153,23 @@ class PointerData {
   }
 }
 
+/// A function used to notify the engine about how a [PointerDataPacket] was used in the framework.
+///
+/// For example, can be used to [preventDefault] the source events on the web.
+typedef _PointerDataPacketAcknowledgeFn = void Function({
```

Comment:
Not sure if this needs to be public so programmers can call the `acknowledge` function.